src/main/java/bio/terra/stairway/RetryRuleFixedInterval.java

```diff
@@ -30,9 +34,16 @@ public void initialize() {
   @Override
   public boolean retrySleep() throws InterruptedException {
     if (retryCount >= maxCount) {
+      logger.info("Retry rule fixed: try {} of {} - not retrying", retryCount, maxCount);
```

Comment:
If the maxCount 3, then it will retry 0, 1, and 2. When it hits 3, it stops. And 4/3 is a surprise.
Zero-based doesn't work with a "N of M" phrasing, does it. I'll make the counter 1-based.